nemo/single_node.yaml

```diff
@@ -0,0 +1,21 @@
+run_name: #run-name-here
+platform: #platform-name-here
+gpu_num: 8
+image: nvcr.io/nvidia/nemo:22.09
+env_variables:
+  - key: PYTHONUNBUFFERED
+    value: '1'
+command: | 
+
+  # getting the vocab, merge files for the tokenizer
+  wget <https://s3.amazonaws.com/models.huggingface.co/bert/gpt2-vocab.json>
+  wget <https://s3.amazonaws.com/models.huggingface.co/bert/gpt2-merges.txt>
+
+  python3 /mnt/nvme/jobs/nemo/NeMo-113-release/examples/nlp/language_modeling/megatron_gpt_pretraining.py \\
```

Comment:
I think these end of lines are supposed to be single slashes?

Reply:
Yeah, good catch sorry I was copying around things (from notion) and it added them...